docs/feeds.md

```diff
@@ -14,41 +14,39 @@ In the *Hooks* pattern, we set up a feed using a [webhook](https://en.wikipedia.
 <!-- The github feed is implemented using webhooks.  Put a link here when we have the open repo ready -->
 
 ## Polling
-In the "Polling" pattern, we arrange for an OpenWhisk *action* to poll an endpoint periodically to fetch new data.
-This pattern is relatively easy to build, but the frequency of events will
-of course be limited by the polling interval.
+In the "Polling" pattern, we arrange for an OpenWhisk *action* to poll an endpoint periodically to fetch new data. This pattern is relatively easy to build, but the frequency of events will of course be limited by the polling interval.
 
 ## Connections
 In the "Connections"  pattern, we stand up a separate service somewhere that maintains a persistent connection to a feed source.    The connection based implementation might interact with a service endpoint via long polling, or to set up a push notification.
 
 <!-- Our cloudant changes feed is connection based.  Put a link here to
 an open repo -->
 
+<!-- What is the foundation for the Message Hub feed? If it is "connections" then lets put a link here as well -->
```

Comment:
@csantanapr - the links above are not samples. From what I understand those are actual implementations of packages that do the integration. 

I wrote a sample that shows how end user can consume messages and publish into Kafka - I added a link to the docs to point to this sample: https://github.com/romankhar/Aqueduct .

I am updating the doc now and will submit a new PR with this link included.